src/components/wizard-navigation-bar.component.html

```diff
@@ -13,11 +13,10 @@
         <ng-container *ngIf="step.stepTitleTemplate" [ngTemplateOutlet]="step.stepTitleTemplate.templateRef"></ng-container>
         <ng-container *ngIf="!step.stepTitleTemplate">{{step.stepTitle}}</ng-container>
       </div>
-      <div *ngIf="step.stepSymbolTemplate" class="step-indicator">
-        <ng-container [ngTemplateOutlet]="step.stepSymbolTemplate.templateRef"></ng-container>
+      <div class="step-indicator" [ngStyle]="{ 'font-family': step.stepSymbolTemplate ? '' : step.navigationSymbol.fontFamily }">
```

Comment:
Shouldn't it be `initial` instead of `''`?

Reply:
'initial' will override user-defined styles, which, I believe, would be an unintended behavior.
Demo: https://stackblitz.com/edit/angular5-small-demos

Reply:
Ok, so basically, if an empty string is passed to a css attribute in `[ngStyle]` it is ignored/removed?

Reply:
This is actually a feature of DOM API. When you assign empty value to el.style._property_, the CSS property is actually removed. Demo: https://jsfiddle.net/tcaz6kqu/

Reply:
Ok, I didn't know that.